config/locales/en.yml

```diff
@@ -204,7 +204,7 @@ en:
     rubygem:
       owners_header: Owners
     destroy:
-      request_queued: Your account deletion request has been enqueued. We will send you a confrimation mail when your request has been processed.
+      request_queued: Your account deletion request has been enqueued. We will send you a confirmation mail when your request has been processed.
```

Comment:
Can you please update following two as well? Also, we will be able to merge this change sooner if you make a seperate PR.
https://github.com/rubygems/rubygems.org/blob/e22d7219d7d0e279742b2b27ce6322194a39ba14/test/functional/profiles_controller_test.rb#L184
https://github.com/rubygems/rubygems.org/blob/b2113ffd7e01114a84f83e7560789496559431bd/test/integration/profile_test.rb#L128

Reply:
OK. Sorry I didn't see those test